annotations-runtime/shared/src/main/java/com/github/dynamicextensionsalfresco/webscripts/AnnotationWebScriptBuilder.java

```diff
@@ -0,0 +1,526 @@
+package com.github.dynamicextensionsalfresco.webscripts;
+
+import com.github.dynamicextensionsalfresco.webscripts.annotations.Attribute;
+import com.github.dynamicextensionsalfresco.webscripts.annotations.Authentication;
+import com.github.dynamicextensionsalfresco.webscripts.annotations.Before;
+import com.github.dynamicextensionsalfresco.webscripts.annotations.Cache;
+import com.github.dynamicextensionsalfresco.webscripts.annotations.ExceptionHandler;
+import com.github.dynamicextensionsalfresco.webscripts.annotations.Transaction;
+import com.github.dynamicextensionsalfresco.webscripts.annotations.Uri;
+import com.github.dynamicextensionsalfresco.webscripts.annotations.WebScript;
+import com.github.dynamicextensionsalfresco.webscripts.arguments.HandlerMethodArgumentsResolver;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+import java.util.LinkedHashSet;
+import java.util.List;
+import kotlin.jvm.internal.Intrinsics;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.springframework.beans.factory.BeanFactory;
+import org.springframework.beans.factory.BeanFactoryAware;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
+import org.springframework.core.annotation.AnnotationUtils;
+import org.springframework.extensions.webscripts.Description;
+import org.springframework.extensions.webscripts.Description.FormatStyle;
+import org.springframework.extensions.webscripts.Description.Lifecycle;
+import org.springframework.extensions.webscripts.Description.RequiredAuthentication;
+import org.springframework.extensions.webscripts.Description.RequiredTransaction;
+import org.springframework.extensions.webscripts.Description.TransactionCapability;
+import org.springframework.extensions.webscripts.DescriptionImpl;
+import org.springframework.extensions.webscripts.TransactionParameters;
+import org.springframework.util.Assert;
+import org.springframework.util.ClassUtils;
+import org.springframework.util.ReflectionUtils;
+import org.springframework.util.StringUtils;
+
+/**
+ * Creates {@link AnnotationWebScript} instances from beans defined in a {@link BeanFactory}.
+ *
+ * @author Laurens Fridael
```

Comment:
Should we remove the author annotations as discussed in other projects?
They tend to get out of date really fast, and give wrong impression of the maintenance ownership.